src/servlet/MyServlet.java

```diff
@@ -0,0 +1,18 @@
+package servlet;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+
+    public class MyServlet extends HttpServlet {
+        @Override
+        protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+            try(OutputStreamWriter writer=new OutputStreamWriter(resp.getOutputStream())){
+                writer.write("Java Web Application");
```

Comment:
Yes, this is one way to get the string printed on screen - by using a servlet. But to be precise, the task required to do this via a static page 
![image](https://user-images.githubusercontent.com/48795460/76831155-d5f03280-6837-11ea-9e50-d0bc1bd3b998.png)